gcloud/dns/changes.py

```diff
@@ -221,7 +221,8 @@ def create(self, client=None):
         self._set_properties(api_response)
 
     def exists(self, client=None):
-        """API call:  test for the existence of the change set via a GET request
+        """API call:  test for the existence of the change set via a GET
+        request.
```

Comment:
Sure, as a general guideline it's fine. But it isn't so universally true to
be considered dogma. So many projects have rightfully broke it in order to
have better docs (flask and requests being notable examples).

On Thu, Aug 18, 2016, 5:59 PM Tres Seaver <notifications@github.com> wrote:

> In gcloud/dns/changes.py
> <https://github.com/GoogleCloudPlatform/gcloud-python/pull/2139#discussion_r75414368>
> :
>
> > @@ -221,7 +221,8 @@ def create(self, client=None):
> >          self._set_properties(api_response)
> >
> >      def exists(self, client=None):
> > -        """API call:  test for the existence of the change set via a GET request
> > +        """API call:  test for the existence of the change set via a GET
> > +        request.
>
> @jonparrott <https://github.com/jonparrott> ISTM that breaking with the
> PEP is just wrong: the "single line summary" discipline *works* as a way
> to focus intent. When I find myself creeping up to the limit, finding a way
> to shorten that line (maybe by moving some content down to a more
> "narrative" paragraph) always improves it.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-python/pull/2139/files/180fa5b7f29f5a3815ea33c927c577565035a8e5#r75414368>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAPUc5lClluv85r2hYoETEJS87Xe9Io2ks5qhP_lgaJpZM4Jn88w>
> .
>